tests/sentry/api/endpoints/test_team_alerts_triggered.py

```diff
@@ -0,0 +1,158 @@
+        assert len(response.data) == 13
+        for i in response.data:
+            assert i["count"] == 1
+
+    def test_not_as_simple(self):
+        team_with_user = self.create_team(
+            organization=self.organization, name="Lonely Team", members=[self.user]
+        )
+
+        project1 = self.create_project(
+            teams=[self.team], slug="foo"
+        )  # This project will return counts for this team
+        project2 = self.create_project(
+            teams=[team_with_user], slug="bar"
+        )  # but not this project, cause this team isn't on it (but the user is)
+
+        user_owned_rule = self.create_alert_rule(
+            organization=self.organization,
+            projects=[project1],
+            name="user owned rule",
+            query="",
+            aggregate="count()",
+            time_window=1,
+            threshold_type=AlertRuleThresholdType.ABOVE,
+            resolve_threshold=10,
+            threshold_period=1,
+            owner=ActorTuple.from_actor_identifier(self.user.id),
+        )
+        user_owned_incident = self.create_incident(
+            projects=[project2], status=20, alert_rule=user_owned_rule
+        )
+        team_owned_rule = self.create_alert_rule(
+            organization=self.organization,
+            projects=[project1],
+            name="team owned rule",
+            query="",
+            aggregate="count()",
+            time_window=1,
+            threshold_type=AlertRuleThresholdType.ABOVE,
+            resolve_threshold=10,
+            threshold_period=1,
+            owner=ActorTuple.from_actor_identifier(f"team:{self.team.id}"),
+        )
+        team_owned_incident = self.create_incident(
+            projects=[project1], status=20, alert_rule=team_owned_rule
+        )
+
+        trigger = self.create_alert_rule_trigger(team_owned_rule, "hi", 100)
+        trigger2 = self.create_alert_rule_trigger(team_owned_rule, "bye", 50)
+        trigger3 = self.create_alert_rule_trigger(user_owned_rule, "meow", 200)
+        trigger4 = self.create_alert_rule_trigger(user_owned_rule, "woof", 200)
+        IncidentTrigger.objects.create(
+            incident=user_owned_incident,
+            alert_rule_trigger=trigger,
+            status=TriggerStatus.RESOLVED.value,
+        )  # Resolved trigger doesn't get counted.
+        IncidentTrigger.objects.create(
+            incident=user_owned_incident,
+            alert_rule_trigger=trigger2,
+            status=TriggerStatus.ACTIVE.value,
+        )
+        IncidentTrigger.objects.create(
+            incident=team_owned_incident,
+            alert_rule_trigger=trigger3,
+            status=TriggerStatus.ACTIVE.value,
+            date_added=before_now(days=2),
+        )
+        IncidentTrigger.objects.create(
+            incident=user_owned_incident,
+            alert_rule_trigger=trigger4,
+            status=TriggerStatus.ACTIVE.value,
+            date_added=before_now(days=4),
+        )
```

Comment:
Same comment about triggers here

Reply:
Removed all the trigger stuff now, leftover from when I was querying triggers